sample/src/main/res/layout/fragment_login.xml

```diff
@@ -1,32 +1,164 @@
+            app:layout_constraintBottom_toTopOf="@id/loginButton"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/summaryTextView" />
+
+        <EditText
+            android:id="@+id/apiKeyEditText"
+            style="@style/LoginEditText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/login_field_api_key_hint"
+            tools:ignore="Autofill" />
+
+        <EditText
+            android:id="@+id/userIdEditText"
+            style="@style/LoginEditText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/login_field_user_id_hint"
+            tools:ignore="Autofill" />
+
+        <EditText
+            android:id="@+id/userTokenEditText"
+            style="@style/LoginEditText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/login_field_user_token"
+            android:singleLine="false"
+            android:inputType="textMultiLine"
+            tools:ignore="Autofill" />
+
+        <EditText
+            android:id="@+id/userNameEditText"
+            style="@style/LoginEditText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/login_field_user_name_hint"
+            tools:ignore="Autofill" />
+
+        <View
+            android:id="@+id/bottomGap"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            app:layout_constraintBottom_toTopOf="@+id/loginButton"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/userNameEditText" />
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/loginButton"
+            style="@style/Widget.MaterialComponents.Button.UnelevatedButton"
+            android:layout_width="match_parent"
+            android:layout_height="48dp"
+            android:layout_marginBottom="4dp"
+            android:insetTop="0dp"
+            android:insetBottom="0dp"
+            android:text="@string/login_button_text"
+            android:textAppearance="@style/text"
+            android:textSize="16sp"
+            app:cornerRadius="@dimen/login_controls_corner_radius"
+            app:layout_constraintBottom_toTopOf="@id/developmentUsers"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent" />
+
+        <TextView
+            android:id="@+id/developmentUsers"
+            android:layout_width="match_parent"
+            android:layout_height="48dp"
+            android:autoLink="web"
```

Comment:
Since this text view is supposed to be clicked by the user it would be nice to provide some basic touch-feedback, e.g.:
```suggestion
            android:autoLink="web"
            android:background="?selectableItemBackground"
```

Reply:
Or is the touch feedback provided by `autoLink="web"`?

Reply:
Replaced `android:autoLink="web"` with `android:background="?selectableItemBackground"`